extensions/multibindings/src/com/google/inject/multibindings/ProvidesIntoOptional.java

```diff
@@ -50,6 +50,9 @@
 @Target(METHOD)
 @Retention(RUNTIME)
 public @interface ProvidesIntoOptional {
+  /**
+   * @since 4.0
```

Comment:
The reason I added them was because they're included in the report that `ant jdiff` spits out. I can remove them from the inner classes if you don't mind the report not having a clean output